queue_job/job.py

```diff
@@ -437,12 +467,10 @@ def description(self):
         else:
             return '%s.%s' % (self.model_name, self.func.__name__)
 
-    @property
+    @cached_truthy_property
```

Comment:
Isn't usually called a lazy property? Couldn't the decorator be `lazy_property`?